### PR TITLE
FS-1878 Lead assessor dashboard

### PR DESCRIPTION
**Pull request by harryyo**
### Lead assessor dashboard metrics added

- replaced `get_todo_summary()` with `get_assessments_stats()` which fetches data from new `/assessments/get-stats/{fund_id}/{round_id}` endpoint on assessment_store
- New template markup and css styles added
- Some commented (duplicated from fsd_utils CommonConfig) development config vars removed
- fsd_utils version updated to 1.0.17
- Tests updated to confirm new feature is functioning
- new sample endpoint data added to endpoint_data and duplicated section removed

### CODE REVIEW

File: app/assess/data.py [hunk 1]
```diff
@@ -249,38 +249,14 @@ def get_applications(params: dict) -> Union[List[Application], None]:
...
-                        for application in applications
-                        if application["status"] == "NOT_STARTED"
-                    ]
-                ),
-            }
-        )
-
-        return todo_summary
-    return None
+def get_assessments_stats(fund_id: str, round_id: str) -> Dict:
```
Comment (tferns) [inline app/assess/data.py:252]:
```suggestion
def get_assessments_stats(fund_id: str, round_id: str) -> Dict | None:
```

Could still return `None` due to how `get_data` works

File: app/static/src/styles/landing.css [hunk 2]
```diff
@@ -39,3 +39,35 @@
...
     font-size: 1.1875rem;
 }
+
+headline-container p {
+    margin-bottom: 3px;
+}
+
+headline-container p {
+    margin-bottom: 3px;
+}
```
Comment (tferns) [inline app/static/src/styles/landing.css:49]: Are these needed?

**Review by tferns — APPROVED** [review]
Great work!

File: app/assess/data.py (hunk 1 above)
Comment (harryyo) [reply app/assess/data.py:252]: Yep you're right, thanks

Comment (harryyo) [reply app/assess/data.py]: Done

File: app/static/src/styles/landing.css (hunk 2 above)
Comment (harryyo) [reply app/static/src/styles/landing.css]: Done

**Pull request merged**